Source/WebKit/WebProcess/Network/WebLoaderStrategy.cpp

```diff
@@ -292,6 +292,10 @@ bool WebLoaderStrategy::tryLoadingUsingURLSchemeHandler(ResourceLoader& resource
     if (!webPage || !webFrame)
         return false;
 
+    auto url = resourceLoader.request().url();
+    if (url.isAboutBlank() || url.isAboutSrcDoc())
```

Comment:
We shouldn't need this code here. If this is ever possible for this condition to be true, then it means you are missing some checks in WebCore/